test/batchedadjtrans.jl

```diff
@@ -0,0 +1,58 @@
+function print_array_strs(x)
+    str = sprint((io, x)->show(io, MIME"text/plain"(), x), x)
+    return @view split(str, '\n')[2:end]
+end
+
+@testset "BatchedAdjOrTrans" begin
+    x = randn(Float32, 3,4,2)
+    y = cu(x)
+
+    bax = batched_adjoint(x)
+    btx = batched_transpose(x)
+    bay = batched_adjoint(y)
+    bty = batched_transpose(y)
+
+    rbax = reshape(bax, :)
```

Comment:
vec?

Reply:
yes?

Reply:
I wanted to put a suggestion in swapping `reshape(..., :)` to `vec`, but GH mobile wouldn't let me. Thought of a better suggesion in the meantime.